llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -505,12 +505,18 @@ class TargetLoweringBase {
     return TypePromoteInteger;
   }
 
-  // Return true if the half type should be passed around as i16, but promoted
-  // to float around arithmetic. The default behavior is to pass around as
-  // float and convert around loads/stores/bitcasts and other places where
-  // the size matters.
+  // Return true if the half type should be promoted using soft promotion rules
+  // where each operation is promoted to f32 individually, then converted to
+  // fp16. The default behavior is to promote links of operations, keeping
```

Comment:
Do you mean "chains of operations"?

Reply:
Yes, thank you. I struggled to find the right wording, managed to remember to update it in the commit message and MR description, but forgot to also update the comment. Fixed.